bees/slackbee/slackbee.go

```diff
@@ -66,7 +66,7 @@ func (mod *SlackBee) Action(action bees.Action) []bees.Placeholder {
 		msgParams := slack.NewPostMessageParameters()
 		msgParams.LinkNames = 1
 		for _, to := range tos {
-			_, _, err := mod.client.PostMessage(to, text, msgParams)
+			_, _, err := mod.client.PostMessage(to, slack.MsgOptionText(text, false))
```

Comment:
A better fix would be to set `msgParams := slack.PostMessageParameters{}`, then `_, _, err := mod.client.PostMessage(to, slack.MsgOptionPostMessageParameters(msgParams))`.

That shouldn't change the behavior.